// node-appwrite/dist/enums/email-template-locale.js
'use strict';

var EmailTemplateLocale = /* @__PURE__ */ ((EmailTemplateLocale2) => {
  EmailTemplateLocale2["Af"] = "af";
  EmailTemplateLocale2["Arae"] = "ar-ae";
  EmailTemplateLocale2["Arbh"] = "ar-bh";
  EmailTemplateLocale2["Ardz"] = "ar-dz";
  EmailTemplateLocale2["Areg"] = "ar-eg";
  EmailTemplateLocale2["Ariq"] = "ar-iq";
  EmailTemplateLocale2["Arjo"] = "ar-jo";
  EmailTemplateLocale2["Arkw"] = "ar-kw";
  EmailTemplateLocale2["Arlb"] = "ar-lb";
  EmailTemplateLocale2["Arly"] = "ar-ly";
  EmailTemplateLocale2["Arma"] = "ar-ma";
  EmailTemplateLocale2["Arom"] = "ar-om";
  EmailTemplateLocale2["Arqa"] = "ar-qa";
  EmailTemplateLocale2["Arsa"] = "ar-sa";
  EmailTemplateLocale2["Arsy"] = "ar-sy";
  EmailTemplateLocale2["Artn"] = "ar-tn";
  EmailTemplateLocale2["Arye"] = "ar-ye";
  EmailTemplateLocale2["As"] = "as";
  EmailTemplateLocale2["Az"] = "az";
  EmailTemplateLocale2["Be"] = "be";
  EmailTemplateLocale2["Bg"] = "bg";
  EmailTemplateLocale2["Bh"] = "bh";
  EmailTemplateLocale2["Bn"] = "bn";
  EmailTemplateLocale2["Bs"] = "bs";
  EmailTemplateLocale2["Ca"] = "ca";
  EmailTemplateLocale2["Cs"] = "cs";
  EmailTemplateLocale2["Cy"] = "cy";
  EmailTemplateLocale2["Da"] = "da";
  EmailTemplateLocale2["De"] = "de";
  EmailTemplateLocale2["Deat"] = "de-at";
  EmailTemplateLocale2["Dech"] = "de-ch";
  EmailTemplateLocale2["Deli"] = "de-li";
  EmailTemplateLocale2["Delu"] = "de-lu";
  EmailTemplateLocale2["El"] = "el";
  EmailTemplateLocale2["En"] = "en";
  EmailTemplateLocale2["Enau"] = "en-au";
  EmailTemplateLocale2["Enbz"] = "en-bz";
  EmailTemplateLocale2["Enca"] = "en-ca";
  EmailTemplateLocale2["Engb"] = "en-gb";
  EmailTemplateLocale2["Enie"] = "en-ie";
  EmailTemplateLocale2["Enjm"] = "en-jm";
  EmailTemplateLocale2["Ennz"] = "en-nz";
  EmailTemplateLocale2["Entt"] = "en-tt";
  EmailTemplateLocale2["Enus"] = "en-us";
  EmailTemplateLocale2["Enza"] = "en-za";
  EmailTemplateLocale2["Eo"] = "eo";
  EmailTemplateLocale2["Es"] = "es";
  EmailTemplateLocale2["Esar"] = "es-ar";
  EmailTemplateLocale2["Esbo"] = "es-bo";
  EmailTemplateLocale2["Escl"] = "es-cl";
  EmailTemplateLocale2["Esco"] = "es-co";
  EmailTemplateLocale2["Escr"] = "es-cr";
  EmailTemplateLocale2["Esdo"] = "es-do";
  EmailTemplateLocale2["Esec"] = "es-ec";
  EmailTemplateLocale2["Esgt"] = "es-gt";
  EmailTemplateLocale2["Eshn"] = "es-hn";
  EmailTemplateLocale2["Esmx"] = "es-mx";
  EmailTemplateLocale2["Esni"] = "es-ni";
  EmailTemplateLocale2["Espa"] = "es-pa";
  EmailTemplateLocale2["Espe"] = "es-pe";
  EmailTemplateLocale2["Espr"] = "es-pr";
  EmailTemplateLocale2["Espy"] = "es-py";
  EmailTemplateLocale2["Essv"] = "es-sv";
  EmailTemplateLocale2["Esuy"] = "es-uy";
  EmailTemplateLocale2["Esve"] = "es-ve";
  EmailTemplateLocale2["Et"] = "et";
  EmailTemplateLocale2["Eu"] = "eu";
  EmailTemplateLocale2["Fa"] = "fa";
  EmailTemplateLocale2["Fi"] = "fi";
  EmailTemplateLocale2["Fo"] = "fo";
  EmailTemplateLocale2["Fr"] = "fr";
  EmailTemplateLocale2["Frbe"] = "fr-be";
  EmailTemplateLocale2["Frca"] = "fr-ca";
  EmailTemplateLocale2["Frch"] = "fr-ch";
  EmailTemplateLocale2["Frlu"] = "fr-lu";
  EmailTemplateLocale2["Ga"] = "ga";
  EmailTemplateLocale2["Gd"] = "gd";
  EmailTemplateLocale2["He"] = "he";
  EmailTemplateLocale2["Hi"] = "hi";
  EmailTemplateLocale2["Hr"] = "hr";
  EmailTemplateLocale2["Hu"] = "hu";
  EmailTemplateLocale2["Id"] = "id";
  EmailTemplateLocale2["Is"] = "is";
  EmailTemplateLocale2["It"] = "it";
  EmailTemplateLocale2["Itch"] = "it-ch";
  EmailTemplateLocale2["Ja"] = "ja";
  EmailTemplateLocale2["Ji"] = "ji";
  EmailTemplateLocale2["Ko"] = "ko";
  EmailTemplateLocale2["Ku"] = "ku";
  EmailTemplateLocale2["Lt"] = "lt";
  EmailTemplateLocale2["Lv"] = "lv";
  EmailTemplateLocale2["Mk"] = "mk";
  EmailTemplateLocale2["Ml"] = "ml";
  EmailTemplateLocale2["Ms"] = "ms";
  EmailTemplateLocale2["Mt"] = "mt";
  EmailTemplateLocale2["Nb"] = "nb";
  EmailTemplateLocale2["Ne"] = "ne";
  EmailTemplateLocale2["Nl"] = "nl";
  EmailTemplateLocale2["Nlbe"] = "nl-be";
  EmailTemplateLocale2["Nn"] = "nn";
  EmailTemplateLocale2["No"] = "no";
  EmailTemplateLocale2["Pa"] = "pa";
  EmailTemplateLocale2["Pl"] = "pl";
  EmailTemplateLocale2["Pt"] = "pt";
  EmailTemplateLocale2["Ptbr"] = "pt-br";
  EmailTemplateLocale2["Rm"] = "rm";
  EmailTemplateLocale2["Ro"] = "ro";
  EmailTemplateLocale2["Romd"] = "ro-md";
  EmailTemplateLocale2["Ru"] = "ru";
  EmailTemplateLocale2["Rumd"] = "ru-md";
  EmailTemplateLocale2["Sb"] = "sb";
  EmailTemplateLocale2["Sk"] = "sk";
  EmailTemplateLocale2["Sl"] = "sl";
  EmailTemplateLocale2["Sq"] = "sq";
  EmailTemplateLocale2["Sr"] = "sr";
  EmailTemplateLocale2["Sv"] = "sv";
  EmailTemplateLocale2["Svfi"] = "sv-fi";
  EmailTemplateLocale2["Th"] = "th";
  EmailTemplateLocale2["Tn"] = "tn";
  EmailTemplateLocale2["Tr"] = "tr";
  EmailTemplateLocale2["Ts"] = "ts";
  EmailTemplateLocale2["Ua"] = "ua";
  EmailTemplateLocale2["Ur"] = "ur";
  EmailTemplateLocale2["Ve"] = "ve";
  EmailTemplateLocale2["Vi"] = "vi";
  EmailTemplateLocale2["Xh"] = "xh";
  EmailTemplateLocale2["Zhcn"] = "zh-cn";
  EmailTemplateLocale2["Zhhk"] = "zh-hk";
  EmailTemplateLocale2["Zhsg"] = "zh-sg";
  EmailTemplateLocale2["Zhtw"] = "zh-tw";
  EmailTemplateLocale2["Zu"] = "zu";
  return EmailTemplateLocale2;
})(EmailTemplateLocale || {});

exports.EmailTemplateLocale = EmailTemplateLocale;
